context.go

```diff
@@ -179,6 +179,10 @@ func (c *Ctx) resolveProjectRoot(path string) (string, error) {
 //
 // The second returned string indicates which GOPATH value was used.
 func (c *Ctx) SplitAbsoluteProjectRoot(path string) (string, error) {
+	if path == filepath.Join(c.GOPATH, "src") {
```

Comment:
We need to be a little tricksier about this, I think:

1. Do the `filepath.Join()` and assign into a var
2. Check if lens of `path` and the new var are the same
2. Compare `path` and new var with `internal.HasFilepathPrefix()`

That should take care of any icky capitalization issues.